### PR TITLE
updated

### PR DESCRIPTION
**Pull request by farihaNaqvi**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <body>
+    <body style="#00a2e8; color:white; text-align:center;">
```
Comment (tlcraft) [inline src/index.html:3]: Thanks for updating the README! 

I see one last issue with this styling. You'll need to be sure to add `background-color` to the style. You added the color itself but missed the attribute name. Try this out:

`<body style="background-color: #00a2e8; color:white; text-align:center;">`

**Comment by farihaNaqvi** [comment]
done

**Review by tlcraft — APPROVED** [review]
Nice work! Thanks for updating those things!

**Comment by farihaNaqvi** [comment]
now can this  link to hacktoberfest???

**Comment by farihaNaqvi** [comment]
thank you sir☺️

**Pull request merged**